### PR TITLE
Added @decode_params(format='json') usage example

### PR DESCRIPTION
**Pull request by sergiobrr**
Added @decode_params(format='json') usage example

### CODE REVIEW

File: tg/decorators.py [hunk 1]
```diff
@@ -329,7 +329,42 @@ class decode_params(object):
 
     By default the arguments are parsed in **JSON** format (which is
     currently the only supported format).
-
+    
+    This should be used like:
+        Image you are posting a payload of type 'application/json' like:
+        
+        ticket = {
```
Comment (amol-) [inline tg/decorators.py]: This should be formatted as code too, you probably want two `::` in the previous chapter and indent accordingly.

File: tg/decorators.py [hunk 2]
```diff
@@ -329,7 +329,42 @@ class decode_params(object):
...
+        
+        .. code-block:: python
+        
+        @expose('json')
+        @decode_params(format='json')
+        def create(self, **kw):
+            print('*' * 60)
+            print('kw', kw)
+            ...do stuff...
+            return dict(ticket=something)
```
Comment (amol-) [inline tg/decorators.py]: I think you can remove this first example and leave around only the second.
We don't have to explain python packing and unpacking of arguments :D

File: tg/decorators.py [hunk 3]
```diff
@@ -329,7 +329,42 @@ class decode_params(object):
...
+          typology: 'Xmas Discount',
+          quantity: 3,
+          price: 4.75
+        }
+        
+        Your controller's method should be something like
+        
+        .. code-block:: python
+        
+        @expose('json')
```
Comment (amol-) [inline tg/decorators.py]: Doesn't this need to be properly indented inside the `.. code-block::`?
See https://github.com/TurboGears/tg2/blob/eb81fd4bf5faa125edf0dfb6fd72407fead7efde/tg/support/registry.py#L77-L88 there the code is over indented from the `.. code-block:: python`

File: tg/decorators.py (hunk 2 above)
Comment (sergiobrr) [reply tg/decorators.py]: As you want, it is the same, it was just one example more

**Pull request merged**